aws/resource_aws_organizations_organization.go

```diff
@@ -221,6 +273,47 @@ func resourceAwsOrganizationsOrganizationUpdate(d *schema.ResourceData, meta int
 		}
 	}
 
+	if d.HasChange("enabled_policy_types") {
+		defaultRootID := d.Get("roots.0.id").(string)
+		o, n := d.GetChange("enabled_policy_types")
+		oldSet := o.(*schema.Set)
+		newSet := n.(*schema.Set)
+
+		for _, v := range oldSet.Difference(newSet).List() {
+			policyType := v.(string)
+			input := &organizations.DisablePolicyTypeInput{
+				PolicyType: aws.String(policyType),
+				RootId:     aws.String(defaultRootID),
+			}
+
+			log.Printf("[DEBUG] Disabling Policy Type in Organization: %s", input)
+			if _, err := conn.DisablePolicyType(input); err != nil {
+				return fmt.Errorf("error disabling policy type (%s) in Organization (%s) Root (%s): %s", policyType, d.Id(), defaultRootID, err)
+			}
+
+			if err := waitForOrganizationDefaultRootPolicyTypeDisable(conn, policyType); err != nil {
+				return fmt.Errorf("error waiting for policy type (%s) disabling in Organization (%s) Root (%s): %s", policyType, d.Id(), defaultRootID, err)
+			}
+		}
+
+		for _, v := range newSet.Difference(oldSet).List() {
+			policyType := v.(string)
+			input := &organizations.EnablePolicyTypeInput{
+				PolicyType: aws.String(policyType),
+				RootId:     aws.String(d.Get("roots.0.id").(string)),
```

Comment:
_minor_ we can use `defaultRootID` here?

Reply:
Yes we can! Thanks!